homework-3/src/Router.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import DoctorProfile from './views/DoctorProfile';
+import PatientProfile from './views/PatientProfile';
+import SignIn from './views/SignIn';
+import SignUp from './views/SignUp';
+import RestorePassword from './views/RestorePassword';
+
+const Router = () => (
+  <Switch>
+    <Route exact path='/'>
```

Comment:
I think hardcoding path url is bad practice 
u can create routes.js file where u can store all your path strings in variables
Example: `export default {
  signupPage: '/',
  signinPage: '/signin',
  restorePage: '/restore',
  confirmPage: '/confirm',
  doctorPage: '/doctor',
  patientPage: '/patient',
}`
After that, u can import routes.js in Router.js component and change `Route exact path={routes.signupPage}`
If something changed in ur project, better to change the path in one variable.

Reply:
Added file `routes`